core/block_svg.js

```diff
@@ -1547,6 +1547,8 @@ Blockly.BlockSvg.prototype.render = function(opt_bubble) {
   } else {
     this.renderInternal();
   }
+  // No matter how we rendered, connections locations should now be correct.
```

Comment:
nit, s/connections/connection/ (no s, since it's modifying locations)

Reply:
Done